Gems/Atom/RHI/DX12/Code/Include/Atom/RHI.Reflect/DX12/PipelineLayoutDescriptor.h

```diff
@@ -38,10 +38,13 @@ namespace AZ
             RootParameterIndex m_resourceTable;
 
             /// The root indices of the SRG unbounded array resource descriptor tables (if any).
-            /// Note that on DX12 we support at most one SRV and one UAV in a Srg.
-            static const uint32_t MaxUnboundedArrays = 2;
+            /// TODO: This restriction should be lifted
```

Comment:
Feel free to create a Jira (assigned to me) and tag it here, otherwise this work will get lost.